diff --git a/node_modules/react-native-svg/src/ReactNativeSVG.ts b/node_modules/react-native-svg/src/ReactNativeSVG.ts
index cc15811..794a956 100644
--- a/node_modules/react-native-svg/src/ReactNativeSVG.ts
+++ b/node_modules/react-native-svg/src/ReactNativeSVG.ts
@@ -24,14 +24,6 @@ import Mask from './elements/Mask';
 import Marker from './elements/Marker';
 import ForeignObject from './elements/ForeignObject';
 import { parse, SvgAst, SvgFromUri, SvgFromXml, SvgUri, SvgXml } from './xml';
-import {
-  SvgCss,
-  SvgCssUri,
-  SvgWithCss,
-  SvgWithCssUri,
-  inlineStyles,
-} from './css';
-import { LocalSvg, WithLocalSvg, loadLocalRawResource } from './LocalSvg';
 import {
   RNSVGCircle,
   RNSVGClipPath,
@@ -87,14 +79,6 @@ export {
   SvgFromXml,
   SvgUri,
   SvgXml,
-  SvgCss,
-  SvgCssUri,
-  SvgWithCss,
-  SvgWithCssUri,
-  inlineStyles,
-  LocalSvg,
-  WithLocalSvg,
-  loadLocalRawResource,
   Shape,
   RNSVGMarker,
   RNSVGMask,
diff --git a/node_modules/react-native-svg/src/index.d.ts b/node_modules/react-native-svg/src/index.d.ts
index 891a99d..e8de3c6 100644
--- a/node_modules/react-native-svg/src/index.d.ts
+++ b/node_modules/react-native-svg/src/index.d.ts
@@ -264,6 +264,7 @@ export type Ellipse = React.ComponentClass<EllipseProps>;
 
 export interface GProps extends CommonPathProps, FontProps {
   opacity?: NumberProp;
+  children?: React.ReactNode;
 }
 export const G: React.ComponentClass<GProps>;
 export type G = React.ComponentClass<GProps>;
